spyder/preferences/general.py

```diff
@@ -199,6 +199,7 @@ def setup_page(self):
                                            "these options (in "
                                            "English).").format(HDPI_QT_PAGE))
         screen_resolution_label.setWordWrap(True)
+        screen_resolution_label.setOpenExternalLinks(True)
```

Comment:
This is a small fix for this dialog> I figured it was small enough to go here

<img width="1041" alt="Screen Shot 2019-05-10 at 10 49 15" src="https://user-images.githubusercontent.com/3627835/57539978-46bde880-7311-11e9-9b6f-5b03ba5f2a55.png">